src/orb.yml.hbs

```diff
@@ -94,6 +94,55 @@ examples:
           jobs:
             - verify-deployment
 
+  run-task-fargate:
+    description: Start the run of an ECS task on Fargate.
+    usage:
+      version: 2.1
+
+      orbs:
+        aws-cli: circleci/aws-cli@0.1.4
+        aws-ecs: circleci/aws-ecs@0.0.12
+
+      jobs:
+        run-task:
+          docker:
+            - image: circleci/python:3.7.1
+          steps:
+            - aws-ecr/run-task:
+                cluster: "cluster1"
+                task-definition: "myapp"
+                awsvpc: true
+                subnet-ids: "subnet-70faa93b,subnet-bcc54b93"
+                security-group-ids: "sg-010a460f7f442fa75"
```

Comment:
🤔 Surely both of these values should be stubbed out and be required to have "real" values injected at runtime?

Reply:
Hmm what do you mean @sgerrand ?

Reply:
👋 Apologies. I should have been clearer in my previous comment.

I meant that the values for the keys in this step (e.g. `subnet-ids` and `security-group-ids`) are using "real" values. Other examples in this file are using variables for substitution–perhaps this should use the same approach?

Reply:
Do you mean for example:

```subnet-ids: $SUBNET_ONE, $SUBNET_TWO```

Reply:
I think the real values are helpful as an example of what typical values would look like. Perhaps something like this would be nice?

```
# e.g. "subnet-70faa93b,subnet-bcc54b93"
subnet-ids: $SUBNET_ONE, $SUBNET_TWO
# e.g. "sg-010a460f7f442fa75"
security-group-ids: $SECURITY_GROUP_IDS
```